ui/app/models/database/role.js

```diff
@@ -91,7 +91,10 @@ export default Model.extend({
 
   get showFields() {
     let fields = ['name', 'database', 'type'];
-    fields = fields.concat(getRoleFields(this.type)).concat(['creation_statements', 'revocation_statements']);
+    fields = fields.concat(getRoleFields(this.type)).concat(['creation_statements']);
+    if (this.database != 'elasticsearch') {
```

Comment:
Although this works, it's best practice to use a strict equals (`===` or `!==`). I also think this would be a great place for a comment so when someone returns to this part of the code, they quickly understand why elasticsearch was excluded. 
```suggestion
// elasticsearch does not support revocation statements: https://www.vaultproject.io/api-docs/secret/databases/elasticdb#parameters-1
    if (this.database[0] !== 'elasticsearch') {
```